### DISCUSSION TITLE
[Done] Update useraccounts:core

### DISCUSSION THREAD

**Discussion by vparpoil** [meteor/meteor#11743] [Compat Packages]
Please update the compat version of useraccounts:core to update the dependency to [softwarerero:meteor-accounts-t9n](https://github.com/softwarerero/meteor-accounts-t9n)
The old dependency has a old coffeescript version that is not compatible with other packages dependencies like cultofcoders:grapher 

References : 
https://github.com/meteor-useraccounts/core/issues/754
https://github.com/meteor-useraccounts/core/pull/763

**Comment by filipenevola** [comment]
Hi @vparpoil I see this PR https://github.com/meteor-useraccounts/core/pull/763/files but I believe it's missing a Npm.depends, what do you think?

And is it really necessary to repeat this:

```js
import { T9n } from 'meteor-accounts-t9n';
T9n.setLanguage('en');
import { en } from 'meteor-accounts-t9n/build/en';
T9n.map("en", en);
```

on every template?

**Reply by vparpoil** [reply]
Hi Filipe, 
Here is a proposition of evolution which adds the NPM dependency inside the package : https://github.com/meteor-compat/useraccounts-core/pull/2

**Comment by vparpoil** [comment]
this can be closed now !
